pkgs/development/compilers/llvm/11/llvm.nix

```diff
@@ -122,7 +122,7 @@ in stdenv.mkDerivation (rec {
     "-DCAN_TARGET_i386=false"
   ] ++ optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
     "-DCMAKE_CROSSCOMPILING=True"
-    "-DLLVM_TABLEGEN=${buildPackages.llvm_10}/bin/llvm-tblgen"
+    "-DLLVM_TABLEGEN=${buildPackages.llvm_11}/bin/llvm-tblgen"
```

Comment:
Can you make this dynamic based on the llvm version? If not please add a comment where the version is defined to avoid such regressions in the future.

Reply:
I think the maintainer did a copy and paste error (10/llvm.nix -> 11/llvm.nix). This issue could be found earlier if we would push cross compilation more.
See also https://github.com/NixOS/nixpkgs/pull/100574